ixwebsocket/IXWebSocketTransport.cpp

```diff
@@ -51,10 +51,22 @@
 #include <thread>
 
 
```

Comment:
indent nitpick / we put curly braces on the next line as a convention in our code base. I'll fix that myself.